ROADMAP.md

```diff
@@ -8,36 +8,23 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.12
+## Rook Ceph 1.13
 
-The following high level features are targeted for Rook v1.12 (July 2023). For more detailed project tracking see the [v1.12 board](https://github.com/rook/rook/projects/29).
+The following high level features are targeted for Rook v1.13 (November 2023). For more detailed project tracking see the [v1.13 board](https://github.com/rook/rook/projects/30).
```

Comment:
```suggestion
The following high level features are targeted for Rook v1.13 (November 2023). For more detailed project tracking see the [v1.13 board(https://github.com/rook/rook/projects/30).
```

Reply:
The link looks correct to me with the `]`

Reply:
@travisn  wrote:
> The link looks correct to me with the `]`

right, my suggestion was meant to be different, just intended to remove the space between `]` and `(` . I fixed the suggestion accordingly ...

Reply:
hmm I don't see a space. Are you just looking at the github UI adding a newline there for the view?

Reply:
@travisn  wrote:
> hmm I don't see a space. Are you just looking at the github UI adding a newline there for the view?


quite possible.
if there isn't a newline in the patch, then maybe some other whitespace which I was suggesting  to remove

Reply:
If I expand my browser, it does not wrap or show any space there

Reply:
I  checked it by locally fetching this PR. there is no whitespace. the link formatting is clean!